install.sh

```diff
@@ -0,0 +1,227 @@
+#!/bin/sh
+# Installs the `fiftyone` package and its dependencies.
+#
+# Usage:
+#   sh install.sh
+#
+# Copyright 2017-2025, Voxel51, Inc.
+# voxel51.com
+#
+
+# Show usage information
+usage() {
+    echo "Usage:  sh $0 [-h] [-b] [-d] [-e] [-m] [-p] [-o]
+
+Getting help:
+-h      Display this help message.
+
+Custom installations:
+-b      Source install of fiftyone-brain.
+-d      Install developer dependencies.
+-e      Source install of voxel51-eta.
+-m      Install MongoDB from scratch, rather than installing fiftyone-db.
+-p      Install only the core python package, not the App.
+-o      Install docs dependencies.
+"
+}
+
+# Parse flags
+SHOW_HELP=false
+SOURCE_BRAIN_INSTALL=false
+DEV_INSTALL=false
+DOCS_INSTALL=false
+SOURCE_ETA_INSTALL=false
+SCRATCH_MONGODB_INSTALL=false
+BUILD_APP=true
+while getopts "hbdempo" FLAG; do
+    case "${FLAG}" in
+        h) SHOW_HELP=true ;;
+        b) SOURCE_BRAIN_INSTALL=true ;;
+        d) DEV_INSTALL=true ;;
+        e) SOURCE_ETA_INSTALL=true ;;
+        m) SCRATCH_MONGODB_INSTALL=true ;;
+        p) BUILD_APP=false ;;
+        o) DOCS_INSTALL=true ;;
+        *)
+            usage
+            exit 1
+            ;;
+    esac
+done
+[ "$SHOW_HELP" = true ] && usage && exit 0
+
+set -e
+NODE_VERSION=22.14.0
+OS=$(uname -s)
+
+MIN_MINOR=9
+MAX_MINOR=12
+
+if command -v python3 >/dev/null 2>&1; then
+    PYTHON=python3
+elif command -v python >/dev/null 2>&1; then
+    PYTHON=python
+else
+    echo "ERROR: Neither python3 nor python found in PATH."
+    exit 1
+fi
+
+# Extract major.minor version info
+PY_VER=$($PYTHON -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")')
+MAJOR=$(echo "$PY_VER" | cut -d. -f1)
+MINOR=$(echo "$PY_VER" | cut -d. -f2)
+
+if [ "$MAJOR" -ne 3 ] || [ "$MINOR" -lt "$MIN_MINOR" ] || [ "$MINOR" -gt "$MAX_MINOR" ]; then
+    echo "Python $PY_VER is NOT supported. Please use Python 3.$MIN_MINOR - 3.$MAX_MINOR."
+    exit 1
+fi
+
+echo "Python $PY_VER is supported."
+
+# Ensure pip targets this Python interpreter
+PIP="$PYTHON_BIN -m pip"
```

Comment:
Is this supposed to be `$PYTHON`?
```suggestion
PIP="$PYTHON -m pip"
```
Without this change I'm getting a failure on line 159 below.